include/RAJA/util/View.hpp

```diff
@@ -68,7 +68,14 @@ struct View {
   {
   }
 
-  RAJA_INLINE constexpr View(View const &) = default;
+  //We found that the generated copy constructor does not actually copy-construct
+  //the object on the device in certain nvcc versions. 
+  //By specifying the copy constructor we are able ensure proper behavior.
```

Comment:
Change "specifying the copy ..." to "explicitly defining the copy..." (see above).